dotCMS/src/main/java/com/dotcms/rest/api/v1/site/SiteResource.java

```diff
@@ -244,4 +275,252 @@ public final Response switchSite(
         }
 
     }
-} // E:O:F:SiteBrowserResource.
\ No newline at end of file
+
+    @GET
```

Comment:
We need curl-tests for all these endpoints and probably IT for testing permissions